app/views/shared/_nav.html.erb

```diff
@@ -2,5 +2,8 @@
 <%= link_to 'Home', root_path %>
 <%= link_to 'Browse Items', items_path %>
 <%= link_to 'View Merchants', merchants_path %>
+<% if current_admin? %>
```

Comment:
This might be better in the  admin nav partial at: `app/views/shared/_admin_nav.html.erb` -- Which will obviate the need for doing another `current_admin?` check

Reply:
Agreed

Reply:
Addressed in new commit